ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -183,8 +157,8 @@ def get_demo_files(path: str) -> List[str]:
 
 @timed
 def load_demonstration(
-    file_path: str
-) -> Tuple[BrainParameters, List[AgentInfoActionPairProto], int]:
+    file_path: str,
```

Comment:
I don't think the comma is needed

Reply:
Required by `black` formatting... I tried 😢